releng/hu.bme.mit.massif.parent/pom.xml

```diff
@@ -50,24 +56,31 @@
 		<!-- Update site -->
 		<module>../../releng/hu.bme.mit.massif.site</module>
 		
+		<!-- CLI jar package -->
+		<module>../../releng/hu.bme.mit.massif.simulink.cli-package</module>
+		
 		<!-- Maven projects -->
 		<module>../../maven/massif-communication-runtime</module>
 		<module>../../maven/massif-matlabcontrol-runtime</module>
 		<module>../../maven/massif-commandevaluation-runtime</module>
+		<module>../../maven/massif-simulink-emf</module>
 
 		
 	</modules>
 	<properties>
 		<tycho.version>1.0.0</tycho.version>
 		<xtend.compiler.version>2.12.0</xtend.compiler.version>
```

Comment:
Make sure a consistent Xtend version is used througout the project

Reply:
OK, I'll update this to 2.13.0